nodejs/webrtc-hello-again/server.js

```diff
@@ -0,0 +1,257 @@
+    res.send('{"status":"Failed to start PSTN call; no session id found"}')
+    return;
+  }
+  participant_response = null;
+  console.log("about to create participant")
+  // create a participant for this browser user
+  var participantBody = new BandwidthWebRTC.Participant(
+    {
+      "tag" : uuid.v1(), 
+      "publishPermissions" : ["AUDIO"]
+    });
+
+  webRTCController.createParticipant(accountId, participantBody, 
+    async function(error, createResponse, context) {      
+      // console.log("PSTN participant response info: ", createResponse)
+      participant_response = createResponse
+      PSTNToken = createResponse.token;
+      
+      var body = new BandwidthWebRTC.Subscriptions({"sessionId" : getSessionId()});
+      console.log("Add PSTN caller to the session")
+      webRTCController.addParticipantToSession(accountId, getSessionId(), createResponse.participant.id, body, 
+        function(error, response, context) {
+          // console.log("addParticipantToSession response:", response)         
+      }).catch(error => {
+        console.error('Error in addParticipantToSession:', error);
+        res.send('{"status":"Failed to add PSTN call to session"}')
+      });
+
+      var body = new BandwidthVoice.ApiCreateCallRequest({
+        "from": process.env.FROM_NUMBER,
+        "to": process.env.OUTBOUND_PHONE_NUMBER,
+        "applicationId": process.env.VOICE_APPLICATION_ID,
+        "answerUrl": process.env.BASE_CALLBACK_URL + 'callAnswered',
+        "answerMethod": "POST",
+        "callTimeout": 10
+      });
+
+      console.log("start the call to", process.env.OUTBOUND_PHONE_NUMBER)
+      try{
+        var response = await voiceController.createCall(accountId, body)
+        callId = response.callId;
+        calls.set(response.callId, participant_response)
+        // console.log("voice API reponse: ", response);      
+        res.send('{"status":"ringing"}')
+  
+      } catch (e) {
+        console.error(`error calling ${process.env.OUTBOUND_PHONE_NUMBER}`, e);
+
+        res.send('{"status":"call failed"}')
+      }
+  }).catch(error => {
+    console.error('Error in createParticipant:', error);
+    res.send('{"status":"Failed to get token"}')
+  });  
+
+
+
+});
+
+/**
+ * Bandwidth's Voice API will hit this endpoint when an outgoing call is answered
+ */
+app.post("/callAnswered", async (req, res) => {
+  // console.log("Callback from Voice system:", req.body)
+  const callId = req.body.callId;
+  console.log(`received answered callback for call ${callId} tp ${req.body.to}`);
+
+  const participant = calls.get(callId);
+  if (!participant) {
+    console.log(`no participant found for ${callId}!`);
+    res.status(400).send();
+    return;
+  }
+
+  // This is the response payload that we will send back to the Voice API to transfer the call into the WebRTC session
+  // const bxml = webRTCController.generateTransferBxml(participant.token);
```

Comment:
Commented code

Reply:
Leaving this one in - we'll be switching over to the SDK shortly. But I added a comment next to this line and the alternate one explaining it.